js/test/unit/vector-tests.ts

```diff
@@ -18,7 +18,7 @@
 import { TextEncoder } from 'text-encoding-utf-8';
 import Arrow from '../Arrow';
 import { type, TypedArray, TypedArrayConstructor, Vector } from '../../src/Arrow';
-import { packBools } from '../../src/util/bit'
```

Comment:
@TheNeuralBit ah, do we not export this util in the main package? if so, we can. that way we'll test the compiled code. On line 20 I think we should also do `const { type, Vector } = Arrow;`, and only import the interfaces from the source dir.

Reply:
Yeah good call. My syntax checker, [tsuquyomi](https://github.com/Quramy/tsuquyomi), complains about the `const { type, Vector } = Arrow;` approach so I shied away from it, but the tests run just fine.